samples/dnn/classification.cpp

```diff
@@ -198,16 +207,39 @@ int main(int argc, char** argv)
         }
         std::string label = format("Inference time of 1 round: %.2f ms", t1);
         std::string label2 = format("Average time of 200 rounds: %.2f ms", timeRecorder.getTimeMilli()/200);
-        putText(frame, label, Point(0, 15), FONT_HERSHEY_SIMPLEX, 0.5, Scalar(0, 255, 0));
-        putText(frame, label2, Point(0, 35), FONT_HERSHEY_SIMPLEX, 0.5, Scalar(0, 255, 0));
+        cv::putText(frame, label, Point(0, 15), FONT_HERSHEY_SIMPLEX, 0.5, Scalar(0, 255, 0));
```

Comment:
why use `cv::` if we already put `using namespace cv;` in the beginning of the sample. Normally, in OpenCV samples we don't use `cv::` to make the code cleaner.